applovin_max/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Versions
 
+## 3.9.0
```

Comment:
It's a patch version bump on the native SDKs, so we can just mirror that in the native plugin (you did that for React Native, why not here)

Reply:
We haven't used patch for a while.  I agree that a patch looks better in this case.